tests/integration/cli/actions/test_select_config_file.py

```diff
@@ -62,6 +57,7 @@ def test_auto_select_config_file(
                                               config_file=str(config_path)) in captured.out
 
 
+@pytest.mark.skip("planned for removal in PR #3382")
```

Comment:
Are the changes for this test needed for v7.1.x? 3382 is set for 7.2.x

Reply:
There appeared to be an unrelated test failure in this test.  I can investigate the issue, but this test is in support of a very specific configuration behavior when the host system has multiple configuration files. Suggestions?

Reply:
Weird that it fails now. In any case, it makes sense not to spend time on it given the specific area of the code, related to multiple configurations which doesn't really apply to ursula anyway, and if the fix isn't obvious.

Reply:
> Weird that it fails now. 

💯 I suspect that it might be a dependency change?  I agree that this test failure is highly contained and unlikely to indicate any real issues running taco nodes.